mackup/applications/psysh.cfg

```diff
@@ -0,0 +1,5 @@
+[application]
+name = PsySH
+
+[configuration_files]
```

Comment:
This should be an `xdg_configuration_files`: https://github.com/lra/mackup/tree/master/doc#add-support-for-an-application-using-the-xdg-directory